docs/sources/companion-plugins/google-drive.mdx

```diff
@@ -94,6 +94,8 @@ Create a project for your app if you don’t have one yet.
 - On the project’s dashboard,
   [enable the Google Drive API](https://developers.google.com/drive/api/v3/enable-drive-api).
 - [Set up OAuth authorization](https://developers.google.com/drive/api/v3/about-auth).
+    - Under scopes, add the `https://www.googleapis.com/auth/drive.readonly` Drive API scope.
+    - Due to this being a sensitive scope Google will ask you to verify your app which can take up to 8 weeks.
```

Comment:
Let's not include delay information as we do not control how long Google process will take (I couldn't find any mention of 8 weeks in their docs). It's probably a better idea to not have this information in our docs and instead link to Google docs.
```suggestion
  - Under scopes, add the `https://www.googleapis.com/auth/drive.readonly` Drive
    API scope.
  - Due to this being a sensitive scope, your app must complete Google's OAuth
    app verification before being granted access. See
    [OAuth App Verification Help Center](https://support.google.com/cloud/answer/13463073)
    for more information.
```